ddraw/IDirect3DDeviceX.cpp

```diff
@@ -2453,11 +2540,99 @@ HRESULT m_IDirect3DDeviceX::DrawIndexedPrimitive(D3DPRIMITIVETYPE dptPrimitiveTy
 		}
 		else
 		{
+			const UINT stride = GetVertexStride(dwVertexTypeDesc);
+
+			// Handle PositionT
+			if (Config.DdrawConvertHomogeneousW && (dwVertexTypeDesc & 0x0E) == D3DFVF_XYZRHW)
+			{
+				D3DFVF_XYZB1;
```

Comment:
Stray symbol

Reply:
Oh okay, I have to check this looks like it should not be there.

Reply:
Fixed in the latest check-in.